packages/experiment-codec/src/blockchain.ts

```diff
@@ -1,11 +1,5 @@
-import {
-  AnyCodec,
-  BytesCodec,
-  createFixedBytesCodec,
-  FixedBytesCodec,
-  Unknown,
-  UnpackResult,
-} from "./base";
+import { AnyCodec, BytesCodec, UnpackResult } from "./base";
+import { createFixedBytesCodec, FixedBytesCodec } from "./base";
```

Comment:
these imports can be merged

Reply:
done, maybe there is a rule for prettier/eslint to auto fix this scenario ?

Reply:
yup, we need to [lint](https://github.com/import-js/eslint-plugin-import) the import